tests/parser.html

```diff
@@ -0,0 +1,124 @@
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>GooseParser Tests</title>
+    <style type="text/css">
+        div {
+            border: 1px solid black;
+        }
+    </style>
+</head>
+<body data-value="1" data-str="str" data-obj="{}">
+<div id="mocha"></div>
+<script src="js/jquery-1.11.3.min.js"></script>
```

Comment:
.npmignore, you should not provide tests through npm, developers should clone your project to collaborate anyway. So exclude the `test` folder too.